chart/config/proxy/nginx.conf

```diff
@@ -32,22 +30,26 @@ http {
 	# Remove nginx server header
 	# http://nginx.org/en/docs/http/ngx_http_core_module.html#server_tokens
 	server_tokens off;
+	more_clear_headers Server;
+
+    # See Move default writable paths to a dedicated directory (#119)
+    # https://github.com/openresty/docker-openresty/issues/119
+    client_body_temp_path /var/run/openresty/nginx-client-body;
+    proxy_temp_path       /var/run/openresty/nginx-proxy;
+    fastcgi_temp_path     /var/run/openresty/nginx-fastcgi;
+    uwsgi_temp_path       /var/run/openresty/nginx-uwsgi;
+    scgi_temp_path        /var/run/openresty/nginx-scgi;
 
 	##
 	# Logging Settings
-    log_format mylogfmt escape=none $log_message_json;
-    access_log /dev/stdout mylogfmt;
-    error_log /dev/stderr;
+	log_format mylogfmt escape=none $log_message_json;
+	access_log /dev/stdout mylogfmt;
+	error_log /dev/stderr info;
 
 	##
 	# Gzip Settings
 	##
-	gzip on;
```

Comment:
On quick scanning this is sth that stood out: Any reason to drop this here, or an unintentional change?

Reply:
The include `include /etc/nginx/lib.gzip-assets.conf;` contains the directive + additional setting like min length